src/test/java/bio/terra/service/filedata/azure/tables/TableDirectoryDaoConnectedTest.java

```diff
@@ -75,13 +83,26 @@ public void setup() throws Exception {
 
   @After
   public void cleanup() throws Exception {
+    try {
+      var iter = directoryEntriesToCleanup.iterator();
+      while (iter.hasNext()) {
+        tableDirectoryDao.deleteDirectoryEntry(tableServiceClient, iter.next());
+      }
```

Comment:
Using a `for` loop seems more idiomatic to me.
```suggestion
      for (String entry : directoryEntriesToCleanup) {
        tableDirectoryDao.deleteDirectoryEntry(tableServiceClient, entry);
      }
```